### PR TITLE
Update README to adhere to name change

### PR DESCRIPTION
**Pull request by sindrenm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# angular-playbook
+# javascript-playbook
 
-A place to define our conventions for working in AngularJS projects
+A place to define our conventions for working with JavaScript.
```
Comment (mathiasbno) [inline README.md:3]: Maybe it should also mention frameworks? So it reads `with JavaScript and related frameworks`.

Comment (sindrenm) [inline README.md:3]: Sure, but that doesn't belong in this PR. :wink:

**Comment by sindrenm** [comment]
Thanks. :smiley:
